src/pages/MetricDetail.svelte

```diff
@@ -176,6 +177,24 @@
       <td>Version</td>
       <td>{metric.version || 0}</td>
     </tr>
+    <tr>
+      <td>BigQuery</td>
+      <td>
+        {#each metric.bigquery_names.stable_ping_table_names as name}
+          <div>
+            In
+            <a href={getMetricBigQueryURL(params.app, name[0])}>{name[1]}</a>
+            {#if name[0] !== 'events'}
```

Comment:
Just realized this breaks down in this case, where an event metric was (at one point) sent in the metrics ping:

https://deploy-preview-283--glean-dictionary-dev.netlify.app/apps/fenix/metrics/downloads_misc-download_added

We probably need to look at the `type` of the event when making this decision, rather than the name.